src/common/syncjournaldb.cpp

```diff
@@ -389,6 +389,7 @@ bool SyncJournalDb::checkConnect()
                         "errorcount INTEGER,"
                         "size INTEGER(8),"
                         "modtime INTEGER(8),"
+                        "contentChecksum TEXT,"
```

Comment:
Can this contain multiple checksums like the other column called like that? if yes, please add a plural "s" so in future we know for sure

Reply:
(maybe i mix this up and multiple checksums was only possible for the server or for the OC-checksum header?)

Reply:
Yes, the db only has one checksum